_posts/2023-02-02-sergi-delgado-watchtowers.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Sergi Delgado - Watchtowers and Lightning Privacy"
+tags:
+  - Watchtowers
+  - Lightning Privacy
+  - Eltoo
+---
+[Sergi Delgado](https://twitter.com/sr_gi) joins us to discuss Watchtowers, his prior work in Bitcoin and Lightning privacy, Python vs. Rust and the impact of Eltoo.
+
+<iframe src="https://anchor.fm/chaincode/episodes/XXX" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>
+
+[anchor.fm](https://anchor.fm/chaincode/episodes/XXX)
+
+We cover:
+
+- Sergi's prior research
+
+- Mapping Network Topology Research (02:30)
+  - [TxProbe paper](https://arxiv.org/pdf/1812.00942.pdf)
+  - [Andrew Miller](https://soc1024.ece.illinois.edu/)
+  - [Coinscope](https://www.cs.umd.edu/projects/coinscope/coinscope.pdf)
+  - [Martin episode](https://podcast.chaincode.com/2022/05/12/martin-zumsande-21-addrrelay) (04:28)
+  - [Block-only connections PR](https://github.com/bitcoin/bitcoin/pull/15759) by Suhas (07:16)
+
+- [An Empirical Analysis of Privacy in the Lightning Network](https://arxiv.org/pdf/2003.12470.pdf) (09:02) 
+
+- [Lightning white paper, 2016](https://lightning.network/lightning-network-paper.pdf)  (09:32)
+- [Security and Privacy of Lightning Network Payments with Uncertain Channel Balances](https://arxiv.org/pdf/2103.08576.pdf) (13:22)
```

Comment:
I think we also mentioned the episode on Probing with Sergei here, we could link.